### PR TITLE
Add QuasiRMS voltage and current single-phase sensors

### PR DESCRIPTION
**Pull request by mestinso**
- Added CurrentQuasiRMSSensor and VoltageQuasiRMSSensor for single-phase systems
- Mirrored the style, formatting, and documentation of the polyphase equivalent sensors
- Updated the documentation for the other quasi single-phase sensors to refer to these sensors

### CODE REVIEW

File: Modelica/Electrical/QuasiStatic/SinglePhase/Sensors/CurrentQuasiRMSSensor.mo [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "Continuous quasi average RMS of current" annotation(
+    Placement(visible = true, transformation(origin = {0, -110}, extent = {{-10, -10}, {10, 10}}, rotation = -90), iconTransformation(origin = {0, -110}, extent = {{-10, -10}, {10, 10}}, rotation = -90)));
+  SI.ComplexCurrent i "Complex current";
+  SI.ComplexVoltage v "Complex voltage";
+equation
+  I = sqrt(i.re^2 + i.im^2);
+  i = pin_p.i;
+  v = pin_p.v - pin_n.v;
+  v = Complex(0,0);
+  annotation (Documentation(info="<html><head></head><body><p style=\"font-size: 12px; font-family: 'DejaVu Sans Mono';\">This sensor determines the continuous quasi&nbsp;<a href=\"Modelica://Modelica.Blocks.Math.RootMeanSquare\">RMS</a>&nbsp;value of a single-phase current system.</p><pre style=\"font-size: 12px;\">  I = abs(i)</pre>
```
Comment (tobolar) [inline Modelica/Electrical/QuasiStatic/SinglePhase/Sensors/CurrentQuasiRMSSensor.mo:14]: Please delete all the html stuff not being used within MSL: `<head>` and `<body>`, and all the paragraph formating. Only limited html subset is commonly used, check `Modelica.UsersGuide.Conventions.Documentation`.

Comment (mestinso) [reply Modelica/Electrical/QuasiStatic/SinglePhase/Sensors/CurrentQuasiRMSSensor.mo]: Should be fixed now, thanks.

File: Modelica/Electrical/QuasiStatic/SinglePhase/Sensors/VoltageQuasiRMSSensor.mo [hunk 2]
```diff
@@ -11,7 +11,14 @@ equation
...
   i = Complex(0,0);
-  annotation (Documentation(info="<html><head></head><body><p style=\"font-family: 'DejaVu Sans Mono'; font-size: 12px;\">This sensor determines the continuous quasi&nbsp;<a href=\"Modelica://Modelica.Blocks.Math.RootMeanSquare\">RMS</a>&nbsp;value of a single-phase voltage system.</p><pre style=\"font-size: 12px;\">  V = abs(v)</pre>
+  annotation (Documentation(info="<html>
+<p>
+This sensor determines the continuous quasi <a href=\"Modelica://Modelica.Blocks.Math.RootMeanSquare\">RMS</a>
+value of a single-phase voltage system.
+</p>
+<pre>
+  V = abs(v)
+</pre>
```
Comment (beutlich) [inline Modelica/Electrical/QuasiStatic/SinglePhase/Sensors/VoltageQuasiRMSSensor.mo:21]:
```suggestion
<blockquote><pre>
V = abs(v)
</pre></blockquote>
```

File: Modelica/Electrical/QuasiStatic/SinglePhase/Sensors/CurrentQuasiRMSSensor.mo [hunk 3]
```diff
@@ -11,7 +11,14 @@ equation
...
   v = Complex(0,0);
-  annotation (Documentation(info="<html><head></head><body><p style=\"font-size: 12px; font-family: 'DejaVu Sans Mono';\">This sensor determines the continuous quasi&nbsp;<a href=\"Modelica://Modelica.Blocks.Math.RootMeanSquare\">RMS</a>&nbsp;value of a single-phase current system.</p><pre style=\"font-size: 12px;\">  I = abs(i)</pre>
+  annotation (Documentation(info="<html>
+<p>
+This sensor determines the continuous quasi <a href=\"Modelica://Modelica.Blocks.Math.RootMeanSquare\">RMS</a>
+value of a single-phase current system.
+</p>
+<pre>
+  I = abs(i)
+</pre>
```
Comment (beutlich) [inline Modelica/Electrical/QuasiStatic/SinglePhase/Sensors/CurrentQuasiRMSSensor.mo:21]:
```suggestion
<blockquote><pre>
I = abs(i)
</pre></blockquote>
```

File: Modelica/Electrical/QuasiStatic/SinglePhase/Sensors/VoltageQuasiRMSSensor.mo [hunk 4]
```diff
@@ -0,0 +1,41 @@
+within Modelica.Electrical.QuasiStatic.SinglePhase.Sensors;
+model VoltageQuasiRMSSensor "Continuous quasi voltage RMS sensor for single-phase system"
+  extends Modelica.Electrical.QuasiStatic.SinglePhase.Interfaces.RelativeSensorElementary;
+  Modelica.Blocks.Interfaces.RealOutput V(unit="V")
+    "Continuous quasi average RMS of voltage" annotation(
+    Placement(visible = true, transformation(origin = {0, -110}, extent = {{-10, -10}, {10, 10}}, rotation = -90), iconTransformation(origin = {0, -110}, extent = {{-10, -10}, {10, 10}}, rotation = -90)));
```
Comment (beutlich) [inline Modelica/Electrical/QuasiStatic/SinglePhase/Sensors/VoltageQuasiRMSSensor.mo:6]:
```suggestion
    Placement(transformation(origin = {0, -110}, extent = {{-10, -10}, {10, 10}}, rotation = -90), iconTransformation(origin = {0, -110}, extent = {{-10, -10}, {10, 10}}, rotation = -90)));
```

File: Modelica/Electrical/QuasiStatic/SinglePhase/Sensors/CurrentQuasiRMSSensor.mo [hunk 5]
```diff
@@ -0,0 +1,40 @@
+within Modelica.Electrical.QuasiStatic.SinglePhase.Sensors;
+model CurrentQuasiRMSSensor "Continuous quasi current RMS sensor for single-phase system"
+  extends Modelica.Electrical.QuasiStatic.SinglePhase.Interfaces.RelativeSensorElementary;
+  Modelica.Blocks.Interfaces.RealOutput I(unit="A")
+    "Continuous quasi average RMS of current" annotation(
+    Placement(visible = true, transformation(origin = {0, -110}, extent = {{-10, -10}, {10, 10}}, rotation = -90), iconTransformation(origin = {0, -110}, extent = {{-10, -10}, {10, 10}}, rotation = -90)));
```
Comment (beutlich) [inline Modelica/Electrical/QuasiStatic/SinglePhase/Sensors/CurrentQuasiRMSSensor.mo:6]:
```suggestion
    Placement(transformation(origin = {0, -110}, extent = {{-10, -10}, {10, 10}}, rotation = -90), iconTransformation(origin = {0, -110}, extent = {{-10, -10}, {10, 10}}, rotation = -90)));
```

**Comment by mestinso** [comment]
> The reason why the polyphase sensors use the term "QuasiRMS" is that they do not calculate a true RMS value, as this is not defined in case of current and voltage asymmetries. However, in case of a symmetrical current and voltage system, each quasi RMS sensor calculates an RMS value, which is then equal to each phase RMS quantity.
> 
> The situation is different in a single-phase system. From the complex current `i` and the complex voltage `v` the RMS current `I` and the RMS voltage `V` can be calculated. These quantities are indeed RMS quantities and not quasi RMS quantities.
> 
> I therefore propose to change the names of the sensors as suggested in [#3865 (comment)](https://github.com/modelica/ModelicaStandardLibrary/issues/3865#issuecomment-921181026) to:
> 
> * `Modelica.Electrical.QuasiStatic.SinglePhase.Sensors.CurrentRMSSensor`
> * `Modelica.Electrical.QuasiStatic.SinglePhase.Sensors.VoltageRMSSensor`
> 
> Please, also update the documentation accordingly.

Should be resolved now.

**Review by christiankral — APPROVED** [review]
Looks all good now

**Review by AHaumer — APPROVED** [review]
Although I don't understand what's the problem with placing a ComplexBlocks.ComplexMath.ComplexToPolar (which would do the job) after the VoltageSensor or CurrentSensor, it's ok. For polyphase sensors it's a little bit more work (already implemented).

**Pull request merged**